rust/json_stdout_object.rs

```diff
@@ -0,0 +1,19 @@
+//cargo-deps: json="0.12.4"
+
+// Script reads string args and transforms into dict
+
+use std::env;
+use json::JsonValue;
+
+extern crate json;
+fn main() {
+    let args: Vec<String> = env::args().collect();
```

Comment:
If we use `.skip(1)` here, we won't need to use `&args[..1]` below, it could just be `for arg in args {`